posixlib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -294,7 +290,7 @@ object pthread {
   @name("scalanative_size_of_pthread_t")
   def pthread_t_size: CSize = extern
 
-  @name("scalanative_pthread_attr_t")
+  @name("scalanative_size_of_pthread_attr_t")
   def pthread_attr_t_size: CSize = extern
```

Comment:
This should be `@name("scalanative_pthread_attr_t_size")` to match our conventions and then the C file too.

If you could do that for all these helpers to make sure they match then I would be fine with these enhancements.